prow/config.yaml

```diff
@@ -98,14 +98,29 @@ branch-protection:
         istio.io:
           branches: *blocked_branches
         operator:
+          required_pull_request_reviews: &tide_reviews
+            require_code_owner_reviews: false
+          restrictions: &tide_restrictions
+            teams:
+              - repo-admins
           branches: *blocked_branches
         cni:
           branches: *blocked_branches
         installer:
           branches: *blocked_branches
         pkg:
+          required_pull_request_reviews: &tide_reviews
+            require_code_owner_reviews: false
+          restrictions: &tide_restrictions
+            teams:
+              - repo-admins
           branches: *blocked_branches
         common-files:
+          required_pull_request_reviews: &tide_reviews
+            require_code_owner_reviews: false
```

Comment:
don't we want codeowners?

Reply:
Yes of course. Cut & paste syndrome.

What I really wanted with this PR is just restricting merges to admins.